tests/components/androidtv/test_media_player.py

```diff
@@ -371,7 +379,7 @@ def _setup(config):
 async def _test_select_source(hass, config0, source, expected_arg, method_patch):
     """Test that the methods for launching and stopping apps are called correctly when selecting a source."""
     config = config0.copy()
-    config[DOMAIN][CONF_APPS] = {"com.app.test1": "TEST 1"}
+    config[DOMAIN][CONF_APPS] = {"com.app.test1": "TEST 1", "com.app.test3": None}
```

Comment:
The `None` would have to change to `""` if you were to revert the change I commented on above